test/PSRP.Tests.ps1

```diff
@@ -0,0 +1,138 @@
+Describe " PowerShell Remoting basic functional tests" -Tag @("CI") {
+        BeforeALL {
+                    $LinuxHostName = $env:LINUXHOSTNAME
+                    $WindowsHostName = $env:WINDOWSHOSTNAME
+                    $passwordString = $env:PASSWORDSTRING
+                    $badUserName = "badUserName"
+                    $badPassword = "badPassword"
```

Comment:
The omicli command is `/opt/omi/bin/omicli ei -u <hostnameOrDomain\\username> -p <password> --auth NegoWithCreds --hostname <hostname>  root/cimv2 Win32_Process`